### PR TITLE
KAA-1079: Remove relative includes

### PR DESCRIPTION
**Pull request by Dubland**
/cc @rasendubi @GamovCoder

### CODE REVIEW

File: client/client-multi/client-c/listfiles/UnitTest.cmake [hunk 1]
```diff
@@ -244,6 +258,8 @@ kaa_add_unit_test(NAME test_kaa_extension_private
 #        test/kaa_test_external.c
 #        DEPENDS
 #        kaac ${OPENSSL_LIBRARIES})
+#        INC_DIRS
+#        test
```
Comment (rasendubi) [inline client/client-multi/client-c/listfiles/UnitTest.cmake]: Forgot to move `)`

**Comment by rasendubi** [comment]
```
src/kaa/gen/kaa_configuration_gen.h:# include "../kaa_common_schema.h"
src/kaa/gen/kaa_configuration_gen.h:# include "../collections/kaa_list.h"
src/kaa/gen/kaa_logging_gen.h:# include "../kaa_common_schema.h"
src/kaa/gen/kaa_logging_gen.h:# include "../collections/kaa_list.h"
src/kaa/gen/kaa_notification_gen.h:# include "../kaa_common_schema.h"
src/kaa/gen/kaa_notification_gen.h:# include "../collections/kaa_list.h"
src/kaa/gen/kaa_profile_gen.h:# include "../kaa_common_schema.h"
src/kaa/gen/kaa_profile_gen.h:# include "../collections/kaa_list.h"
src/kaa/platform/ext_log_storage.h:#include "../kaa_error.h"
src/kaa/platform-impl/cc32xx/cc32xx_sock.h:#include "../common/kaa_htonll.h"
src/kaa/utilities/kaa_mem.h:#include "../utilities/kaa_log.h"

src/kaa/gen/kaa_configuration_gen.c:# include "../platform/stdio.h"
src/kaa/gen/kaa_configuration_gen.c:# include "../avro_src/avro/io.h"
src/kaa/gen/kaa_configuration_gen.c:# include "../avro_src/encoding.h"
src/kaa/gen/kaa_configuration_gen.c:# include "../utilities/kaa_mem.h"
src/kaa/gen/kaa_logging_gen.c:# include "../platform/stdio.h"
src/kaa/gen/kaa_logging_gen.c:# include "../avro_src/avro/io.h"
src/kaa/gen/kaa_logging_gen.c:# include "../avro_src/encoding.h"
src/kaa/gen/kaa_logging_gen.c:# include "../utilities/kaa_mem.h"
src/kaa/gen/kaa_notification_gen.c:# include "../platform/stdio.h"
src/kaa/gen/kaa_notification_gen.c:# include "../avro_src/avro/io.h"
src/kaa/gen/kaa_notification_gen.c:# include "../avro_src/encoding.h"
src/kaa/gen/kaa_notification_gen.c:# include "../utilities/kaa_mem.h"
src/kaa/gen/kaa_profile_gen.c:# include "../platform/stdio.h"
src/kaa/gen/kaa_profile_gen.c:# include "../avro_src/avro/io.h"
src/kaa/gen/kaa_profile_gen.c:# include "../avro_src/encoding.h"
src/kaa/gen/kaa_profile_gen.c:# include "../utilities/kaa_mem.h"
src/kaa/platform-impl/Econais/EC19D/econais_ec19d_kaa_client.c:#include "../../../platform/kaa_client.h"
test/test_kaa_notification.c:#define KAA_STATUS_FILE_NAME      "./kaa_status.bin"

templates/event/kaa_event_families.hvm:# include "../kaa_event.h"
templates/event/kaa_event_families.hvm:# include "../kaa_error.h"
templates/event/kaa_event_families.cvm:# include "../avro_src/avro/io.h"
templates/event/kaa_event_families.cvm:# include "../kaa_common.h"
templates/event/kaa_event_families.cvm:# include "../kaa_event.h"
templates/event/kaa_event_families.cvm:# include "../kaa_error.h"
templates/event/kaa_event_families.cvm:# include "../utilities/kaa_mem.h"
```

**Comment by rasendubi** [comment]
```
./src/kaa/platform/time.h:33:#include "../platform-impl/posix/posix_time.h"
```

**Comment by forGGe** [comment]
This branch has conflicts that must be resolved
Please, rebase onto latest develop branch

**Comment by rasendubi** [comment]
After merging with develop:

```
./src/kaa/platform-impl/Econais/EC19D/econais_ec19d_kaa_client.c:77:#include "../../../platform/kaa_client.h"
./src/kaa/utilities/kaa_log.h:29:#include "../kaa_error.h"
./src/kaa/utilities/kaa_base64.c:26:#include "../kaa_common.h"
./src/kaa/utilities/kaa_buffer.h:21:#include "../kaa_error.h"
./src/kaa/utilities/kaa_buffer.c:21:#include "../kaa_common.h"
./src/kaa/utilities/kaa_base64.h:27:#include "../kaa_error.h"
./src/kaa/utilities/kaa_mem.c:18:#include "../utilities/kaa_mem.h"
./src/kaa/utilities/kaa_mem.h:24:#include "../utilities/kaa_log.h"
./src/kaa/utilities/kaa_log.c:21:#include "../kaa_common.h"
./src/kaa/utilities/kaa_log.c:22:#include "../platform/ext_system_logger.h"
```

**Comment by forGGe** [comment]
Again there are some conflicts. Rebase required.

**Pull request merged**